app/com/arpnetworking/metrics/portal/scheduling/JobExecutorActor.java

```diff
@@ -140,49 +164,159 @@ public Receive createReceive() {
                 .match(Tick.class, message -> {
                     _periodicMetrics.recordCounter("job-executor-actor-ticks", 1);
 
-                    final JobRepository<T> repo = _jobRef.getRepository(_injector);
-                    final Optional<Job<T>> job = _jobRef.get(_injector);
-                    if (!job.isPresent()) {
-                        LOGGER.warn()
-                                .setMessage("no such job")
-                                .addData("jobRef", _jobRef)
+                    if (!_cachedJob.isPresent()) {
+                        LOGGER.info()
```

Comment:
This should never happen right? Or conversely, should we wait to schedule the timer until after initialization? (the isPresent check is probably still required but at that point becomes and error)